docs/products/redis/concepts/redis-ccr.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Aiven for Redis®* cross-cluster replication
+sidebar_label: Redis CCR
+pro: true
+---
+
+Aiven for Redis®* Cross-Cluster Replication (CCR) feature replicates data from a primary to a replica cluster across various DNS, clouds, or regions, improving data availability and aiding in disaster recovery.
+
+## Features and benefits
+
+- **High availability:** By replicating data across clusters in different locations,
+  Aiven for Redis CCR ensures that your application remains available even if one
+  cluster goes down.
+- **Disaster recovery:** Automatic promotion of the follower cluster to leader status in
+  case of the primary/main cluster's failure provides a robust disaster
+  recovery mechanism.
+- **Geographical distribution:** With the ability to work across all clouds and regions,
+  Aiven for Redis CCR keeps data closer to your users, reducing latency and improving
+  the user experience.
+
+## How CCR works
+
+Aiven for Redis uses an **active-passive replication model** for its CCR feature,
+ensuring high availability and data resilience across deployments.
+
+**Replication process:** The replication process involves two services - the primary and
+replica services. The system automatically and asynchronously copies any new data you
+add to the primary service to the replica service. The primary service manages all
+read and write operations for optimal performance and data consistency under
+normal conditions.
+
+**Handling failures:** In case of any failure of the primary service, Aiven provides
+automatic failover functionality, which promotes the replica service to become
+the new primary, maintaining service continuity.
+
+However, to fully resume normal operations, you must manually update your application’s
+connection settings to point to the new primary service.
+
+## Limitations
+
+- This feature requires `[Pro Platform](/docs/platform/concepts/pro-platform)`.
+- Aiven for Redis CCR follows an active-passive model, which means that only the data
```

Comment:
it's actually not possible to write data in the replica service at all while the integration is active.

Reply:
Just to confirm, when you say “it’s not possible to write data in the replica service during active integration”, are you suggesting that in Aiven for Redis CCR’s active-passive model, all new data is added to the primary service first and then replicated to the replica service? Could you please confirm or provide more details if necessary?

Reply:
how about rewording it to this: 

Aiven for Redis CCR uses an active-passive model where data can be written only to the primary service and then replicated to the replica service.

Reply:
Yes. And the rewrite looks good.